internal/collector/telemetry/process_telemetry.go

```diff
@@ -83,17 +84,27 @@ var viewCPUSeconds = &view.View{
 // NewProcessMetricsViews creates a new set of ProcessMetrics (mem, cpu) that can be used to measure
 // basic information about this process.
 func NewProcessMetricsViews(ballastSizeBytes uint64) *ProcessMetricsViews {
-	return &ProcessMetricsViews{
+	pmv := &ProcessMetricsViews{
 		ballastSizeBytes: ballastSizeBytes,
 		views:            []*view.View{viewAllocMem, viewTotalAllocMem, viewSysMem, viewCPUSeconds},
 		done:             make(chan struct{}),
 	}
+
+	// procfs.Proc is not available on windows and expected to fail.
+	pid := os.Getpid()
+	proc, err := procfs.NewProc(pid)
+	if err == nil {
```

Comment:
Should we check for windows specific error here with the error message or type (if provided)?

Reply:
It is just `errorString` with a very generic error message: `could not read /proc: CreateFile /proc: The system cannot find the file specified.`